prompt-api-playground/README.md

```diff
@@ -0,0 +1,21 @@
+# ✨ Prompt API Playground
+
+I've created a Chrome [Prompt API Playground](https://tomayac.github.io/prompt-api-playground/) that hopefully is useful to more people:
```

Comment:
nit: who's "I" in this context? Maybe "The Chrome team has create the Prompt API Playground so people can easily try out the API with different prompts and easily share their work"?

Reply:
Made it clear who am I